tests/cxx/typeid_on_type.cc

```diff
@@ -0,0 +1,36 @@
+//===--- typeid_on_type.cc - test input file for iwyu ---------------------===//
```

Comment:
Ah, you can put both `typedid(type)` and `typeid(expr)` into the same file, makes it a little easier to reason about. I suggest `typeid.cc` like you had before, maybe with two functions and a little comment to describe what the difference is between the two forms.

Reply:
Gotcha, thanks!